docs/sources/mimir/configure/configure-native-histograms-ingestion.md

```diff
@@ -7,11 +7,34 @@ weight: 160
 
 # Configure native histograms
 
-To enable support for ingesting Prometheus native histograms over the [remote write API]({{< relref "../references/http-api#remote-write" >}}) endpoint, set the flag `-ingester.native-histograms-ingestion-enabled=true` on ingesters.
+Prometheus native histograms ingestion is an **experimental** feature of Grafana Mimir.
 
-To enable support for querying native histograms together with [Grafana Mimir query sharding]({{< relref "../references/architecture/query-sharding" >}}), set the flag `-query-frontend.query-result-response-format=protobuf` on query frontends.
+You can configure native histograms ingestion over the Prometheus [remote write API]({{< relref "../references/http-api#remote-write" >}}) endpoint globally or per tenant.
 
-> **Note:** Native histograms is an experimental feature of Grafana Mimir.
+> **Note:** To enable support for querying native histograms together with [Grafana Mimir query sharding]({{< relref "../references/architecture/query-sharding" >}}), make sure that the flag `-query-frontend.query-result-response-format` is set to its default `protobuf` value on query frontends.
```

Comment:
Notes follow the syntax at https://grafana.com/docs/writers-toolkit/write/shortcodes/#example.